tests/phpunit/base-class.php

```diff
@@ -1,10 +1,17 @@
 <?php
 namespace Elementor\Testing;
 
+use Elementor\Plugin;
 use Elementor\Testing\Traits\Base_Elementor;
 use Elementor\Testing\Traits\Extra_Assertions;
 
 class Elementor_Test_Base extends \WP_UnitTestCase {
-
 	use Base_Elementor, Extra_Assertions;
+
+	public function tearDown() {
+		parent::tearDown();
+
+		Plugin::$instance->documents->restore_document();
```

Comment:
It needs to reset the `switched_data` & `current_doc`. Maybe try to re-initialize the manager:
```suggestion
		Plugin::$instance->documents = new Documents_Manager();
```

Reply:
Try it, and some errors fail.